### ISSUE TITLE
Training on Mask R-CNN Memory Out

### ISSUE THREAD

**Issue by engineer1109** [tensorpack/tensorpack#821]
Python 3.6
Tensorflow 1.9rc2
CPU memory 16G    GPU Titan Xp 12G

CPU memory is out on init.

**Comment by ppwwyyxx** [comment]
16G is too small. You can modify `data.py` to use smaller `nr_proc` or `buffer_size` to reduce memory use. http://tensorpack.readthedocs.io/modules/dataflow.html#tensorpack.dataflow.MultiProcessMapDataZMQ

**Comment by engineer1109** [comment]
@ppwwyyxx , Thanks it works.
Do you replace all BatchNormalization with GroupNormalization?

**Comment by ppwwyyxx** [comment]
`config.py` has the options about GroupNorm. Refer to those docs for details.
